src/anomalib/pipelines/tiled_ensemble/components/utils/helper_functions.py

```diff
@@ -0,0 +1,179 @@
+"""Helper functions for the tiled ensemble training."""
+import json
+
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from pathlib import Path
+
+from jsonargparse import ArgumentParser, Namespace
+from lightning import Trainer
+
+from anomalib.data import AnomalibDataModule, get_datamodule
+from anomalib.models import AnomalyModule, get_model
+from anomalib.utils.normalization import NormalizationMethod
+
+from . import NormalizationStage
+from .ensemble_engine import TiledEnsembleEngine
+from .ensemble_tiling import EnsembleTiler, TileCollater
+
+
+def get_ensemble_datamodule(args: dict, tiler: EnsembleTiler, tile_index: tuple[int, int]) -> AnomalibDataModule:
+    """Get Anomaly Datamodule adjusted for use in ensemble.
+
+    Datamodule collate function gets replaced by TileCollater in order to tile all images before they are passed on.
+
+    Args:
+        args: tiled ensemble run configuration.
+        tiler (EnsembleTiler): Tiler used to split the images to tiles for use in ensemble.
+        tile_index (tuple[int, int]): Index of the tile in the split image.
+
+    Returns:
+        AnomalibDataModule: Anomalib Lightning DataModule
+    """
+    data_config = args["data"]
```

Comment:
Minor comment but should we directly pass `args["data"]"` to this method? So, the signature can be `data_args: dict`.